src/components/common/forms/ContactForm.tsx

```diff
@@ -21,14 +24,13 @@ const ContactForm = () => {
   };
 
   const isFormValid = !isSubmitted || isValid;
+  const errorMessage = String(Object.values(errors)[0]?.message || "");
```

Comment:
```suggestion
  const errorMessage = String(Object.values(errors)[0]?.message ?? "");
```
You can use the `??` operator in these cases

Reply:
Ohh, I didn't know. Thanks! Updated